atom/browser/api/lib/web-contents.js

```diff
@@ -70,6 +70,9 @@ let wrapWebContents = function(webContents) {
   var controller, method, name, ref1;
   webContents.__proto__ = EventEmitter.prototype;
 
+  // webContents.debugger is an EventEmitter.
+  webContents.debugger.__proto__ = EventEmitter.prototype;
```

Comment:
This is going to initialize the `debugger` for every `webContents` no matter whether users use it, this doesn't take much resources but is not good. Can you use the `_setWrap` trick to do make it inherit `EventEmitter`?